src/scipp/core/__init__.py

```diff
@@ -21,14 +21,7 @@ def custom_formatwarning(msg, *args, **kwargs):
 # Import errors
 from .cpp_classes import BinEdgeError, BinnedDataError, CoordError, \
                          DataArrayError, DatasetError, DimensionError, \
-                         DTypeError, SizeError, SliceError, \
-                         UnitError, VariableError, VariancesError
-
-BinEdgeError.__doc__ = 'Inappropriate bin-edge coordinate.'
-CoordError.__doc__ = 'Inappropriate coordinate values.'
-UnitError.__doc__ = 'Inappropriate unit value.'
-DimensionError.__doc__ = 'Inappropriate dimension labels and/or shape.'
-DTypeError.__doc__ = 'Inappropriate dtype.'
```

Comment:
I suppose you tested if this still works when set in C++?

Reply:
Yes. `help` and `sphinx` pick them up correctly.